src/actions/vstudio/vs2010_nuget.lua

```diff
@@ -54,33 +37,148 @@
 		end
 	end
 
+	function nuget2010.packageAPIInfo(prj, package)
+		local id = nuget2010.packageId(package)
+		local version = nuget2010.packageVersion(package)
 
---
--- Generates the packages.config file.
---
+		if not packageAPIInfos[package] then
+			local packageAPIInfo = {}
 
-	function nuget2010.generatePackagesConfig(obj)
-		local wks = obj.workspace
+			printf("Examining NuGet package '%s'...", id)
+			io.flush()
 
-		local done = {}
-		local packages = {}
-		for prj in p.workspace.eachproject(wks) do
-			for i = 1, #prj.nuget do
-				local package = prj.nuget[i]
+			local response, err, code = http.get(string.format("https://api.nuget.org/v3/registration1/%s/index.json", id:lower()))
```

Comment:
API's and support aside, let's at least make this configurable... a nuget.serverlist = {}, or something...

Then as for the filelist.. for now, we can simple error... and if we hit it, we can look at implementing whatever is needed to get it to work... Internally here at Blizzard we are using NuGet.Server, so yeah, we're going to hit that, since those implementations only implement the V2 API of nuget, and it seems like you are querying the V3 API.. Which is all right, if we need it, we'll either get to it ourselves, or someone else who needs it will...